### PR TITLE
GameDB: Japanese & Korean disc-additions by arapapa.

### PR DESCRIPTION
**Pull request by pgert**
Submission of Japanese & Korean disc-additions made by arapapa in the forum.
Some name & regional corrections were also made.
No gamefixes added or altered.

### CODE REVIEW

**Comment by pgert** [comment]
I have made sure that none of these new disc are already in the database.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
...
+---------------------------------------------
+Serial = SLKA-25145
+Name   = .hack//Shinshoku Osen Vol. 3
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25174
+Name   = .hack//Zettai Houi Vol. 4
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-15023
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: This one is not in order, should be early in the list. All of the games are in order by serial number and letter prefixes.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
...
+Name   = Assault Suits Valken
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25010
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Same as before.

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
...
+---------------------------------------------
+Serial = SLPM-64551
+Name   = Shin Contra
+Region = NTSC-K
+---------------------------------------------
+Serial = SLPM-64513
+Name   = Crash Bandicoot: Return of The Demon King
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25155
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

File: bin/GameIndex.dbf [hunk 4]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
...
+Name   = Crimson Sea 2
+Region = NTSC-K
+---------------------------------------------
+Serial = SLPM-64532
+Name   = Digital Holmes
+Region = NTSC-K
+---------------------------------------------
+Serial = SCKA-20068
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.
Check the rest as well, don't wanna add more spam comments for the same issue.

**Comment by pgert** [comment]
What difference does it make?

**Comment by pgert** [comment]
The location of the entries does not have any relevance on whether they are in ORDER.
If you want to move stuff around then please do so in a PR own your own.
I don't support your argument.

**Comment by lightningterror** [comment]
Considering all others have been added in numerical order it would be the logical way to do it.
These will look out of place.
Well whatever, your decision.

**Comment by pgert** [comment]
Specify to were you want the entries to be moved.

**Comment by lightningterror** [comment]
Just put them in numerical order, also make sure the letters before the numbers match as well.

File: bin/GameIndex.dbf [hunk 5]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
 Name   = Homura [Taito the Best]
 Region = NTSC-J
 ---------------------------------------------
+Serial = SLKA-15023
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Same issue like the previous.

File: bin/GameIndex.dbf [hunk 6]
```diff
@@ -43673,3 +43673,1123 @@ Serial = TCPS-10172
...
+---------------------------------------------
+Serial = SLKA-25010
+Name   = Chou Battle Houshin
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25155
+Name   = Crimson Sea 2
+Region = NTSC-K
+---------------------------------------------
+Serial = SLKA-25080
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

**Comment by lightningterror** [comment]
Here is an example how to do it.
SLKA-20000
SLKA-20001
SLKA-20002
SLKA-20003
SLKA-20004
SLPM-20000
SLPM-20001
SLPM-20002
SLPM-20003

And so on.

**Comment by pgert** [comment]
I'm not going to put these entries in numerical order.
Thanks for the attention.

**Pull request closed without merging**